docs/articles/nunit/release-notes/framework.md

```diff
@@ -51,6 +54,17 @@ None
 
 ### Others
 
+* [4765](https://github.com/nunit/nunit/issues/4765) Document ThrowOnEachFailurUnderDebugger.
```

Comment:
Question: This would be a documentation update to the docs repo in support of https://github.com/nunit/nunit/issues/4738 above. Do we also track these as framework release notes?

Reply:
Not sure.   I would prefer to have these issues in the docs repo, but up to now we have had them in the code repos.  There are area labels in the docs repo that could be used to point to the right source.  
Since I generate the release notes and it has the milestone, it is picked up.